### PR TITLE
docker_setup #826 without sarus for testing

### PR DESCRIPTION
**Pull request by jdahm**
Uses `scripts/build-and-test` for the test outside a virtual environment.

### CODE REVIEW

**Comment by jdahm** [comment]
launch jenkins

**Comment by jdahm** [comment]
launch jenkins

**Comment by jdahm** [comment]
launch jenkins

**Comment by jdahm** [comment]
launch jenkins

**Comment by jdahm** [comment]
launch jenkins

**Comment by jdahm** [comment]
launch jenkins

**Comment by jdahm** [comment]
@twicki - this is ready to go. I hope to have the actual Docker setup PR soon too!

File: scripts/jenkins/env_daint.sh [hunk 1]
```diff
@@ -8,13 +8,12 @@ module load cray-python/3.6.5.7
 module load cudatoolkit/10.1.105_3.27-7.0.1.1_4.1__ga311ce7
 module load /project/c14/install/daint/clang/module_6.0.1-gcc-8.3.0
 
-export CXX=`which g++`
-export CC=`which gcc`
+export CXX=$(which g++)
```
Comment (jdahm) [inline scripts/jenkins/env_daint.sh:11]: My editor converted these.

File: scripts/jenkins/srun_dawn_PR.sh [hunk 2]
```diff
@@ -1,18 +1,47 @@
...
+
+# Clone clang-gridtools here because the image does not have the ssh key
+if [ -z ${CLANG_GRIDTOOLS_REPOSITORY+x} ]; then
+    CLANG_GRIDTOOLS_REPOSITORY=ssh://git@ssh.github.com:443/MeteoSwiss-APN/clang-gridtools.git
+fi
+if [ -z ${CLANG_GRIDTOOLS_BRANCH+x} ]; then
+    CLANG_GRIDTOOLS_BRANCH=master
+fi
+
+git clone --depth 1 -b $CLANG_GRIDTOOLS_BRANCH $CLANG_GRIDTOOLS_REPOSITORY $root_dir/clang-gridtools
```
Comment (twicki) [inline scripts/jenkins/srun_dawn_PR.sh:22]: is the login node mounted when we are on the compute node?

**Comment by jdahm** [comment]
Closing this as the other PR was merged.

**Pull request closed without merging**